manifests/params.pp

```diff
@@ -48,9 +48,8 @@
   $repo_export_dir = '/var/lib/pulp/katello-export'
 
   # OAUTH settings
-  $oauth_key = 'katello'
-  $oauth_token_file = 'katello_oauth_secret'
-  $oauth_secret = cache_data('foreman_cache_data', $oauth_token_file, random_password(32))
+  $candlepin_oauth_key = cache_data('foreman_cache_data', 'candlepin_oauth_key', random_password(32))
```

Comment:
Do we really want to set this to random? It is kinda nice that the key is set to `katello` as it clearly indicates who is using this to talk to CP.

Reply:
@ehelms The Foreman modules use a random string, though I do tend to agree especially as we move towards split deployments.

I'll change it.

https://github.com/theforeman/puppet-foreman/blob/master/manifests/params.pp#L220
https://github.com/theforeman/puppet-foreman_proxy/blob/master/manifests/params.pp#L359